icicle/include/icicle/utils/utils.h

```diff
@@ -34,6 +34,6 @@ static void print_bytes(const std::byte* data, const uint nof_elements, const ui
       std::cout << std::hex << std::setw(2) << std::setfill('0')
                 << static_cast<int>(data[element_idx * element_size + byte_idx]);
     }
-    std::cout << std::dec << ",\n";
+    std::cout << ",\n";
```

Comment:
why? this makes all future prints be in hex

Reply:
It's not me.....
Didnt touch it